modules/lang-painless/src/main/java/org/elasticsearch/painless/node/ELambda.java

```diff
@@ -100,7 +101,7 @@ void extractVariables(Set<String> variables) {
 
     @Override
     void analyze(Locals locals) {
-        final Definition.Type returnType;
+        final Type returnType;
         final List<String> actualParamTypeStrs;
```

Comment:
Isolated, for this specific file, it would definitely be a poor trade-off.  For the project as it stands right now, I think this is the correct one since the majority (all?) of node files have the static import as Definition.Type.  I believe it's confusing when switching between nodes (at least for me it was).  Longer term, though, I'm 100% on board with renaming the classes in Painless to get conflict-free names.  It definitely sucks having both a Painless Type and and ASM Type.  Originally there was a bit more separation when writing was in a different set of files from analysis.  For now, I'm going to leave this, and let's rename Definition.Type in the next refactor PR which should probably address more pieces of Definition as well.  I believe Uwe had some suggestions there.